_unittest/test_01_Design.py

```diff
@@ -172,17 +172,17 @@ def test_15a_duplicate_design(self):
     def test_15b_copy_design_from(self):
         origin = os.path.join(self.local_scratch.path, "origin.aedt")
         destin = os.path.join(self.local_scratch.path, "destin.aedt")
-        self.aedtapp.save_project(project_file=origin)
+        self.aedtapp.save_project(project_path=origin)
```

Comment:
This should not be file_name as in other refactoring? @Samuelopez-ansys what do you think?